doc/source/examples/devportal.rst

```diff
@@ -0,0 +1,41 @@
+.. _ref_devportal_examples:
+
+
+Developer portal articles
+=========================
+
+In this section, you'll find informative and insightful blog
+articles crafted by our expert developers at Ansys and published
+in the `Developer Portal <developer_portal_>`_.
+Whether you're a seasoned pro or just getting started with PYMAPDL, our blog
+articles cover a wide range of topics, from best practices and tips to in-depth
+tutorials and real-world use cases. Our aim is to provide you with a valuable
+resource that will help you harness the full potential of PYMAPDL for your
+engineering and simulation needs.
+
+If you have questions or suggestions regarding other Ansys products, you can
+also visit the `Developer Portal <developer_portal_>`_. It's your hub for all
+things related to our products, where you can connect with our experts and
+engage with our growing community.
```

Comment:
```suggestion
Because the Developer Poral is your hub for all things related to Ansys
products, if you have questions or suggestions, you can go there to
connect to Ansys experts and the growing Ansys developer community.
```